src/cuda_utils.cc

```diff
@@ -258,6 +263,255 @@ SupportsIntegratedZeroCopy(const int gpu_id, bool* zero_copy_support)
   return Status::Success;
 }
 
+Status
+GetAllocationGranularity(size_t& aligned_sz)
+{
+  CUmemAllocationProp prop = {};
+  prop.type = CU_MEM_ALLOCATION_TYPE_PINNED;
+  prop.location.type = CU_MEM_LOCATION_TYPE_DEVICE;
+  prop.location.id = 0;
+
+  RETURN_IF_ERROR(CudaDriverHelper::GetInstance().CuMemGetAllocationGranularity(
+      &aligned_sz, &prop, CU_MEM_ALLOC_GRANULARITY_MINIMUM));
+  return Status::Success;
+}
+
+CudaDriverHelper::CudaDriverHelper()
+{
```

Comment:
initialize dl_open_handle_ to nullptr for completeness

Reply:
fixed.